### PR TITLE
SOV-3161 Increase borrowing debt fix

### PR DESCRIPTION
**Pull request by tjcloa**
The fix allows to increase debt of existing loans using the loans' existing collateral only.  
The contract LoanOpenings has also been refactored due to the `stack too deep` issue.

### CODE REVIEW

**Review by ororopickpocket — APPROVED** [review]

File: contracts/modules/LoanOpenings.sol [hunk 1]
```diff
@@ -445,6 +437,42 @@ contract LoanOpenings is
...
     }
 
+    function _updateCollateralAfterTrade(
+        bytes32 loanId,
+        LoanParams memory loanParamsLocal,
+        MarginTradeStructHelpers.SentAddresses memory sentAddresses,
+        MarginTradeStructHelpers.SentAmounts memory sentValues,
+        bytes memory loanDataBytes
+    ) internal returns (MarginTradeStructHelpers.SentAmounts memory) {
+        // bytes32 loanIdLocal = loanId;
```
Comment (stoatnfi) [inline contracts/modules/LoanOpenings.sol:447]: Can we remove the commented one?

Comment (cwsnt) [inline contracts/modules/LoanOpenings.sol:447]: can remove the commented code here?

**Review by cwsnt — APPROVED** [review]
Approved with minor comment

File: contracts/modules/LoanOpenings.sol (hunk 1 above)
Comment (tjcloa) [reply contracts/modules/LoanOpenings.sol:447]: removed in this (commit)[https://github.com/DistributedCollective/Sovryn-smart-contracts/commit/30972ee37f140c050116180057818eeb97a6a356]

**Pull request merged**